src/yolo/yolo.jl

```diff
@@ -484,7 +517,7 @@ end
 Reduces the size of array and only keeps detections over threshold
 """
 function keepdetections(arr::AbstractArray)
-    return arr[:, arr[5, :] .> 0]
+    return arr[:, arr[end-2, :] .> 0]
```

Comment:
This appears to be a bug.. The detect thresholding should be on the class confidence
```
class_confidence = objectness_score × class_probability
```
i.e. in terms of indexes
```
[end-2] = [5] x max([6:end-3])
```
So we should be using `end-2`